# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/values.h.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We just place a stub values.h in the dietlibc include directory to get stuff
like popt to compile.

  - Rene Rebe <rene@exactcode.de>

--- dietlibc-0.27/include/values.h.orig	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.27/include/values.h	2005-06-02 17:45:44.051302108 +0200
@@ -0,0 +1,10 @@
+#ifndef _VALUES_H_
+#define _VALUES_H_
+
+#include <stdint.h>
+#include <float.h>
+
+#define MAXINT		INT_MAX
+#define MAXFLOAT	FLT_MAX
+
+#endif /* _VALUES_H_ */
